modules/layers/src/bitmap-layer/bitmap-layer.js

```diff
@@ -29,10 +29,9 @@ const {fp64LowPart} = fp64;
 import vs from './bitmap-layer-vertex';
 import fs from './bitmap-layer-fragment';
 
-const DEFAULT_TEXTURE_PARAMETERS = {
-  [GL.TEXTURE_MIN_FILTER]: GL.LINEAR_MIPMAP_LINEAR,
-  // GL.LINEAR is the default value but explicitly set it here
-  [GL.TEXTURE_MAG_FILTER]: GL.LINEAR,
+export const DEFAULT_TEXTURE_PARAMETERS = {
+  [GL.TEXTURE_MIN_FILTER]: GL.NEAREST,
```

Comment:
I would say yes. You'll likely see textures looking pixelated, more artifacts at acute angles and it might take a performance hit for bigger textures (because of the mipmaps).

I'm not sure what you mean by handling the parameter fallback.

Would it make sense to have different defaults for NPOT textures?

Reply:
Would it make sense for `Texture2D` to automatically disable certain parameters if the size is NPOT?

Reply:
It looks like it should already be doing that: https://github.com/uber/luma.gl/blob/master/modules/webgl/src/classes/texture.js#L673

Maybe that logic's not working?

Reply:
So when we first create the texture, we don't have the dimensions yet (video may still be loading). We just set `width: 1, height: 1` so the default NPOT logic does not kick in. When we call `resize` later the parameters are never checked again.

Reply:
luma does check and replace the parameters for NPOT, if I understand correctly. So here in BitmapLayer, I can keep the original params.